### PR TITLE
feat: added xdcapothem demo file.

### PR DESCRIPTION
**Pull request by AnilChinchawale**
## Summary

added xdcapothem demo file.

## Changes

added xdcapothem.tt file to view demo interface.

### CODE REVIEW

**Comment by osslgtm** [comment]
@AnilChinchawale doesn't seem to be working when i click and drag the demo docs for verification

**Comment by vrushalipanchal** [comment]
@osslgtm, I have tested the shared demo files for both network(XDC Mainnet & XDC Apothem), and they're working fine on my side.

**XDC Apothem:**
https://github.com/TradeTrust/tradetrust-website/blob/2af099ad5673763236c243fce3237b427e176a39/public/static/demo/xdcapothem.tt
image

**XDC Mainnet:**
https://github.com/TradeTrust/tradetrust-website/blob/875ae6ed13523500a273192331f027c0e3ffe400/public/static/demo/xdc.tt
image

**Comment by osslgtm** [comment]
> @osslgtm, I have tested the shared demo files for both network(XDC Mainnet & XDC Apothem), and they're working fine on my side.

yes, but you need to make sure that the demo file interaction is working for this to be considered usable

image

File: src/constants/chain-info.ts
Comment (AnilChinchawale) [inline src/constants/chain-info.ts:1]: @osslgtm We have updated networkName in chain-info to fix the error in the demo file.

**Review by osslgtm — APPROVED** [review]
lgtm

**Pull request merged**